guix-science/packages/bioinformatics.scm

```diff
@@ -74,34 +74,37 @@
 (define-public spades
   (package
    (name "spades")
-   (version "3.15.2")
-   (source (origin
-            (method url-fetch)
-            (uri (string-append
-                  "http://cab.spbu.ru/files/release"
-                  version "/SPAdes-" version ".tar.gz"))
-            (sha256
-             (base32 "03cxz4m1n4rc81lqb4p1pz2ammms7f31wvi4daywfkc13aal6fz9"))))
+   (version "3.15.5")
+   (source
+    (origin
+     (method git-fetch)
+     (uri (git-reference
+           (url "https://github.com/ablab/spades")
+           (commit (string-append "v" version))))
+     (file-name (git-file-name name version))
+     (sha256
+      (base32
+       "0f905nlp8dp295ypa0vlxyb9h0r04x56ykvv630cjl04kd5nx052"))))
    (build-system cmake-build-system)
    ;; Reported under section 2 "Installation", "SPAdes requires a 64-bit
    ;; system": http://cab.spbu.ru/files/release3.10.1/manual.html
    (supported-systems '("x86_64-linux"))
    (arguments
-    `(#:tests? #f ; There is no test target.
+    `(#:tests? #f                       ; There is no test target.
       #:phases
-      (modify-phases %standard-phases
-        (add-before 'configure 'move-to-source-dir
+        (modify-phases %standard-phases
+          (add-before 'configure 'move-to-source-dir
           (lambda _
-            (chdir "src"))))))
+          (chdir "assembler/src"))))))
```

Comment:
Indentation here is not correct.
`(modify` should be right underneath `#:phases` and `(chdir` should be indented to the `a` of `(lambda`.

(When using Emacs it's just `M-q`, provided you're using the scheme-mode with the Guix settings.)